mozetl/clientsdaily/rollup.py

```diff
@@ -0,0 +1,121 @@
+import datetime as DT
+from pyspark.sql import SparkSession
+from pyspark.sql import functions as F
+import click
+from moztelemetry.standards import filter_date_range
+from mozetl.utils import generate_filter_parameters, format_spark_path
+from fields import MAIN_SUMMARY_FIELD_AGGREGATORS
+
+ACTIVITY_SUBMISSION_LAG = DT.timedelta(10)
+MAIN_SUMMARY_VERSION = 4
+MAIN_SUMMARY_PATH = "s3://telemetry-parquet/main_summary/v{}".format(
+    MAIN_SUMMARY_VERSION)
+WRITE_VERSION = '2'
+STORAGE_BUCKET = 'net-mozaws-prod-us-west-2-pipeline-analysis'
+STORAGE_SUB_DIR = '/spenrose/clients-daily/v{}/'.format(WRITE_VERSION)
+
+
+def load_main_summary(spark):
+    return spark.read.option(
```

Comment:
Formatting nit: I prefer using parenthesis to create a new scope where the method operator can work on new lines. For examples:

```
return (
    spark
    .read
    .option("mergeSchema", "true")
    .parquet(MAIN_SUMMARY_PATH)
)
```

If you are vehemently opposed to parentheses, I would suggest indenting the parenthesis after `"mergeSchema", "true"` to the next line.